FlorenceBML/BMLsmarco741/BMLsmarco741.xml

```diff
@@ -58,6 +58,33 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 ይደልዎ፡ የዐትብ፡ ካህን፡ ዐ፡ ዲበ፡ ነፍሱ፡ ዐ፡ ዲበ፡ ሕዝብ፡ ዐዲበ፡ ላእካን፡ ናአኵተከ፡ እግዚኦ፡ በፍቁር፡ ወልድከ፡ ኢየሱስ፡ ክርስቶስ፡ ዘበደኀሪ፡ መዋዕል፡ ፈኖከ፡ ለነ፡ መድኅን፡</incipit>
                         </msItem>
                     </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="17">
+                                    <dimensions unit="mm">
+                                        <height>142</height>
+                                        <width>108</width>                                        
+                                    </dimensions>
+                                    <note>Not certain from the account of the <bibl><ptr target="bm:Marrassini1986Laurenziana"/><citedRange unit="page">103</citedRange>
+                                    </bibl>, whether dimensions refer to height and width of book cover or folia.</note>
+                                    <ab type="pricking">Pricking is visible</ab>
+                                    <ab type="ruling">Ruling is partly visible</ab>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">15th century</seg>
+                                <date notBefore="1400" notAfter="1499">Dated according to paleography</date>
```

Comment:
Here in `handDesc `it should be
```
<seg type="script">Fine (??? or regular, or irregular, or skilled or whatever)</seg>
                                <date notBefore="1400" notAfter="1499">15th century</date>
```
as to "dated according to palaeography" this information should be encoded as an attribute to the date in the `history origin origDate` section
```
<history>
                        <origin><origDate notBefore="1400" notAfter="1499" evidence="lettering"></origDate></origin>                        
                    </history>
```